frontend/components/Auction.vue

```diff
@@ -1,17 +1,21 @@
 <template>
     <TextBlock :title="`Auction #${auctionId}`">
-        <Alert v-if="auctionError" class="my-3" :message="auctionError" type="error" />
+        <div class="my-3">
+            <Alert v-if="auctionError && auctionError.showBanner" :message="auctionError.error" type="error" />
+        </div>
         <div v-if="auction">
-            <RestartBlock
-                v-if="auctionError === 'This auction is inactive and must be restarted'"
+            <AuctionRestartPanel
```

Comment:
I think to set rounded corners on the bottom, this panel should be set into its own `div`